resources/views/components/select.blade.php

```diff
@@ -38,6 +38,7 @@ class="mb-1"
             class="cursor-pointer overflow-hidden !text-transparent !dark:text-transparent"
             x-ref="input"
             x-on:click="toggle"
+            x-on:focus="toggle"
```

Comment:
Using the toggle in the focus event will stop opening the select when clicking on it.
The focus event should open the popover.

Reply:
Should I set popover to true on the focus, or remove the focus instruction on the input in the toggle method ?

Reply:
Yes, just call the open method.